fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalOlapScan.java

```diff
@@ -51,6 +52,8 @@
  */
 public class LogicalOlapScan extends LogicalCatalogRelation implements OlapScan {
 
+    public final TableSample tableSample;
```

Comment:
Optional. put it as last var of this class

Reply:
Declare public fields after privete is not allowed by our checkstyle rules